### ISSUE TITLE
How can i arm and disarm ?

### ISSUE THREAD

**Issue by Ibrahimox** [ptz0n/homebridge-verisure#23]
Only things this done is temp status for me, how can i arm and disarm ? 
Also: Smartcamera was not supported.

**Comment by dhogborg** [comment]
There is experimental support for arm/disarming the alarm (and more!) in this fork of the plugin: https://github.com/dhogborg/homebridge-verisure/

I've been using this without issue for quite some time now, though the typescript variant that I just published is quite young and only used on test bench. I will post updates to this conversation when it matures.

For the impatient, here is a preview: https://www.npmjs.com/package/homebridge-verisure-ts

**Comment by ZEEN0j** [comment]
I got it working and see all my units temperatures using npm install -g homebridge-verisure-ts. But how do I arm and disarm?

**Comment by Merlinuf** [comment]
Well everything works for me but nor ARM/DISARM its not even showing up!
Please help!
What serial should be inserted here ignore_alarms": ["00000000000"]

**Comment by hoefs** [comment]
@ZEEN0j & @Merlinuf & @dhogborg any update on this? How do we get our alarm visible?

**Comment by Merlinuf** [comment]
Nope, i have every accessorys but not the arm/disarm cant get why

**Comment by Merlinuf** [comment]
And What does the ignore alarm do?

**Comment by hoefs** [comment]
Same for me. Everything shows up, but I can't see alarm/disarm.
